BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -114,29 +114,40 @@ void BSP_CAN_Init(callback_t rxEvent, callback_t txEnd, bool loopback) {
     gRxMessage.DLC = 0;
     gRxMessage.FMI = 0;
 
-    /* Enable FIFO 0 message pending Interrupt */
-    CAN_ITConfig(CAN1, CAN_IT_FMP0, ENABLE);
+    /* Enable interrupts if in normal state */
+    if(!Fault_Flag){
```

Comment:
It would be better if BSP_CAN_Init() has another parameter that is like `bool false` passed in as CANBUS_Init already has that bool than using a global flag for fault. Idk if using a global flag for fault would result in any race conditions if we initialize CAN before the fault flag could be set to 1

Reply:
fixed